### DISCUSSION TITLE
Airflow login password encryption

### DISCUSSION THREAD

**Discussion by wangrenz** [apache/airflow#12656] [General] [answered]
Hi:

I add user and passwd by `airflow create_user` for sign in.

But, The result of software capture is plain text.

How to encrypt the password transmission? No https, Thanks.

Sincerely
Renzhi

**Comment by XD-DENG** [comment]
@wangrenz may you please provide more details?  Because the issue doesn't make sense to me (or maybe I misunderstood something).

In the latest 1.10.*, if you are using db for user management, the invoking chain is `airflow create_user` -> `appbuilder.sm.add_user()` -> `airflow.www_rbac.security. AirflowSecurityManager` -> `flask_appbuilder.security.sqla.manager. SecurityManager ` -> then use `werkzeug.security.generate_password_hash` to hash the password. I don't think the password will be stored in DB as plain text.

May you clarify what you meant by "the result of software capture"?

**Comment by XD-DENG** [comment, marked as answer]
I cross-checked with Issue list, and seems @kaxil has already answered this question there and has marked it closed, https://github.com/apache/airflow/issues/12662.

Will mark this discussion as answered as well.
